src/main/java/com/revature/models/RequestComparator.java

```diff
@@ -0,0 +1,25 @@
+package com.revature.models;
```

Comment:
I think it might be better for this class to be in a different package. It isn't exactly a `model` so to speak.

Reply:
I wasn't really sure where to put it, so I stuck it in the Models package so it was together with the model it was comparing. I've moved it to the services package along with classes that will actually use it instead.